test/model/test_deepar_smoke.py

```diff
@@ -0,0 +1,179 @@
+# Standard library imports
+from random import randint
+from typing import List, Tuple
+
+# Third-party imports
+import pytest
+
+# First-party imports
+from gluonts.dataset.common import ListDataset
+from gluonts.model.deepar import DeepAREstimator
+from gluonts.transform import FieldName
+from gluonts.trainer import Trainer
+
+NUM_TS = 5
+START = "2018-01-01"
+FREQ = "D"
+MIN_LENGTH = 20
+MAX_LENGTH = 50
```

Comment:
Sorry for this cherrypicking but if this test is not very fast, can you reduce min_length/max_length to smaller values?